server/webapp/WEB-INF/vm/admin/admin-tab.vm

```diff
@@ -1,130 +0,0 @@
-#*************************GO-LICENSE-START*********************************
```

Comment:
Since this file is deleted, we can also delete `admin-tab.xml` and  its reference from `spring-tabs-servlet.xml`